packages/frontend/src/components/common/modal/DashboardUpdateModal.tsx

```diff
@@ -62,14 +64,35 @@ const DashboardUpdateModal: FC<DashboardUpdateModalProps> = ({
     });
 
     return (
-        <Modal
-            title={<Title order={4}>Update Dashboard</Title>}
+        <MantineModal
+            title="Update Dashboard"
             {...modalProps}
+            icon={IconLayoutDashboard}
```

Comment:
You don't have to do it now but check this out: https://github.com/lightdash/lightdash/issues/14520